llm-monitoring-main.ipynb

```diff
@@ -62,36 +62,21 @@
    "execution_count": null,
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    func.set_image_pull_configuration(image_pull_policy="Always")
</pre><p>Why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demo-monitoring-and-feedback-loop/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>

Reply:
<p>see the above comment</p>